grammars/gfm.cson

```diff
@@ -1002,7 +1002,28 @@
         'include': 'source.clojure'
       }
     ]
-  },
+  }
+  {
+    'begin': '^\\s*(`{3,}|~{3,})\\s*(?=\\s*[-\\w]+\\s*$)'
```

Comment:
Maybe add a comment above this line to make it very clear that this is a "catch-all" for unrecognized languages.  Perhaps `# Attempt to provide syntax highlighting for unknown languages`